torch/export/_trace.py

```diff
@@ -1071,6 +1088,7 @@ def forward(self, *args, **kwargs):
         assert out_spec is not None
 
         gm = ep_non_strict.gm
+        gm.meta["disable_forced_specializations"] = _disable_forced_specializations
```

Comment:
Where's this used? Is this left over from a previous commit?

Reply:
Yeah, sorry, need to clean this up too.